src/crimson/os/seastore/linked_tree_node.h

```diff
@@ -1007,6 +1014,20 @@ class ChildNode : public BaseChildNode<ParentT, key_t> {
   const T& down_cast() const {
     return *static_cast<const T*>(this);
   }
+  btreenode_pos_t get_parent_pos() const {
+    auto &me = down_cast();
+    auto parent = this->get_parent_node();
+    assert(parent);
+    //TODO: can this search be avoided?
+    auto key = me.get_begin();
+    auto iter = parent->lower_bound(key);
+    if (iter.get_key() > key) {
+      assert(iter != parent->end());
+      iter--;
+    }
```

Comment:
Is this an independent fix that should be split out of a clean up?

Reply:
Um, not really, this is a change caused by the need from the omap tree

Reply:
(unrelated to this PR)

Not sure if we can finally drop `fixed_kv_node_meta_t::begin` and `end` to be consistent with omap and onode trees.